content/cfp/CFP Review Board/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: CFP Review Board
+type: cfp
+menu:
+  main:
+    weight: 35
+
+draft: false
+
+---
+
+{{% hero %}}
+
+BSidesTLV awesome CFP review board members.
+
+{{% /hero %}}
+
+{{< teams types="cfp=CFP" >}}
```

Comment:
Not sure we need a dedicated page for this

Why not add this to the existing team page?

Reply:
will be too crowded. I prefer to separate the core team and the cfp team